controllers/grid/users/reviewer/form/ReviewerForm.inc.php

```diff
@@ -241,12 +242,16 @@ function fetch($request) {
 			'reviewDueDate' => __('reviewer.submission.reviewDueDate'),
 			'submissionReviewUrl' => __('common.url'),
 			'reviewerUserName' => __('user.username'),
+			'contextName' => $context->getLocalizedName(),
```

Comment:
I'd suggest having the `emailVariables` describe the contents rather than reiterating them. Think...

> Dear `Recipient Name`,

...rather than...

> Dear `John Doe`,

For the implementation, I don't love static functions, but I also don't love instantiating and populating a MailTemplate object in the `display` function. Perhaps add a static function called e.g. `getEmailVariables` in ArticleMailTemplate, then this class could augment them e.g. something like:

    $templateMgr->assign('emailVariables',
        array_merge(
            ArticleMailTemplate::getEmailVariables(),
            array(
 		$templateMgr->assign('emailVariables', array(
 			'reviewerName' => __('user.name'),
                ...
            )
        )
    );

Does that make sense? Suggestions for improvement are welcome, of course.